src/Html.php

```diff
@@ -210,11 +211,13 @@ public static function cleanParametersURL($url)
      **/
     public static function resume_text($string, $length = 255)
     {
-        if (Toolbox::strlen($string) > $length) {
-            $string = Toolbox::substr($string, 0, $length) . "&nbsp;(...)";
+        $append = '';
+        if (mb_strlen($string, 'UTF-8') > $length) {
```

Comment:
Any reason not to use `Toolbox::strlen()` sinc eit does exactly the same?

Reply:
I am not a big fan of methods that exists just to not have to write a second parameter.

Reply:
OK. So maybe it should be deprecated and replaced then. But out of current scope obviously.